include/configs/warp7.h

```diff
@@ -22,20 +22,20 @@
 #define CONFIG_SYS_FSL_ESDHC_ADDR       USDHC3_BASE_ADDR
 #define CONFIG_SUPPORT_EMMC_BOOT
 #define CONFIG_SYS_FSL_ESDHC_HAS_DDR_MODE
-#define CONFIG_SYS_MMC_IMG_LOAD_PART	1
+#define CONFIG_SYS_MMC_IMG_LOAD_PART	2
 
 #define CONFIG_DFU_ENV_SETTINGS \
 	"dfu_alt_info=boot raw 0x2 0x400 mmcpart 1\0" \
 
 #define CONFIG_EXTRA_ENV_SETTINGS \
 	CONFIG_DFU_ENV_SETTINGS \
 	"script=boot.scr\0" \
-	"image=zImage\0" \
+	"image=/boot/zImage\0" \
```

Comment:
This change the way we use the images. 

I understand why you are doing that but it seems like you will need to have a patch for this. However, we can reduce the changes need using a bootdir variable or similar so you reduce the conflict risk.

Reply:
Thanks Otavio I understand, the best way to handle this integration is to work directly into the distro layer (https://github.com/WaRP7/meta-warp7-distro ), I think ? and keep this configuration ;)

Sorry for the noise ;)

---
Pierre-Jean